processors/base/helpers/__init__.py

```diff
@@ -86,6 +86,10 @@ def clean_identifiers(identifiers):
     }
     result = {}
     for key, value in identifiers.items():
+        try:
+            value = re.sub(r'\s', '', value)
+        except TypeError:
+            pass
```

Comment:
This is kinda hacky, but you could then change to something like `re.sub(r'^([^\s+])\s+', r'\g<1>', value)` instead. This removes the whitespaces between the `NCT` and whatever comes afterwards. To allow us to debug any issue, you can use:

```python
new_value, num_changes = re.subn(r'^([^\s+])\s+', r'\g<1>', value)
if num_changes:
    logger.debug('Removed whitespaces from identifier "%s" to "%s"', value, new_value)
    value = new_value